### PR TITLE
Validation in text & style editor added Ref. #411

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -184,6 +184,8 @@ import { AdbutlerComponent } from './components/adbutler/adbutler.component';
 import { ConsoleService } from './services/settings/console.service';
 import { DefaultConsoleService } from './services/settings/default-console.service';
 import { NodeCreatedLabelStylesFixer } from './components/project-map/helpers/node-created-label-styles-fixer';
+import { NonNegativeValidator } from './validators/NonNegativeValidator';
```
Comment (ziajka) [inline src/app/app.module.ts]: NonNegativeValidator -> non-negative-validator

File: src/app/app.module.ts [hunk 2]
```diff
@@ -184,6 +184,8 @@ import { AdbutlerComponent } from './components/adbutler/adbutler.component';
...
+import { RotationValidator } from './validators/RotationValidator';
```
Comment (ziajka) [inline src/app/app.module.ts]: Same as above

**Comment by ziajka** [comment]
@piotrpekala7 newly added text has no border size assigned, or when null is assigned then requires user to type in border width:
Screenshot_2019-06-04_11-49-05

Expected behavior: 0 value is set when border width is undefined.

File: src/app/app.module.ts (hunk 2 above)
Comment (piotrpekala7) [reply src/app/app.module.ts]: Done

File: src/app/app.module.ts (hunk 1 above)
Comment (piotrpekala7) [reply src/app/app.module.ts]: Done

**Pull request merged**